updater/spec/support/dummy_pkg_helpers.rb

```diff
@@ -89,10 +89,8 @@ def deprecated?
 
     sig { override.returns(T::Boolean) }
     def unsupported?
-      # Check if the feature flag for Bundler v1 unsupported error is enabled.
-      return false unless name == "bundler" && Dependabot::Experiments.enabled?(:bundler_v1_unsupported_error)
-
-      version < supported_versions.first
```

Comment:
Since bundler_v1 unsupported error is activated and stable we don't need feature flag to check it anymore.